microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -295,70 +570,187 @@ DevTools assigns scripts random colors.  In the previous figure, function reques
+
+To add a script to the ignore list, right-click a script in the chart and select **Add script to ignore list**.
+
+<!-- ![The context menu with the ignore script option focused](todo: ignore-script.png) -->
+
+The chart collapses ignored scripts, marks them as **On ignore list**, and adds them to the **Custom exclusion** rules in **Customize and control DevTools** (![The Customize and control DevTools icon](./reference-images/customize-and-control-devtools-icon.png)) > **Settings** > **Ignore list**.  Ignored scripts are saved until you remove them either from the trace or from the **Custom exclusion rules**.
+
+<!-- ![The ignore script list tab in Settings](todo: ignore-script-list.png) -->
 
 
 <!-- ------------------------------ -->
 #### View activities in a table
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#activities -->
 
 After recording a page, in addition to the **Main** section to analyze activities, DevTools also provides three tabular views for analyzing activities.  Each view gives you a different perspective on the activities:
 
-*  To view the activities where the most time was directly spent, use the [Bottom-Up](#the-bottom-up-tab) tab.
+*  To view the activities where the most time was directly spent, use the [Bottom-up](#the-bottom-up-tab) tab.
+
+*  To view the root activities that cause the most work, use the [Call tree](#the-call-tree-tab) tab.
 
-*  To view the root activities that cause the most work, use the [Call Tree](#the-call-tree-tab) tab.
+*  To view the activities in the order in which they occurred during the recording, use the [Event log](#the-event-log-tab) tab.
 
-*  To view the activities in the order in which they occurred during the recording, use the [Event Log](#the-event-log-tab) tab.
+To help you find what you are looking for faster, all three tabs have buttons for advanced filtering next to the **Filter** bar:
 
+* ![The Match case icon](./reference-images/match-case-icon.png) **Match case**.
 
+* ![The Regular expression icon](./reference-images/regex-icon.png) **Regular expression**.
+
+
+<!-- ![The three buttons for advanced filtering](todo: advanced-filtering.png) -->
+
+Each tabular view in the **Performance** tool shows links for activities such as functions calls.  To help you debug, DevTools finds the corresponding function declarations in source files.  Additionally, if the appropriate source maps are present and enabled, DevTools automatically finds the original files.
+
+Click a link to open a source file in the **Sources** tool.
+
+<!-- ![Link to a source file in the Event Log tab](todo: link-a-source-file.png) -->
+
+<!-- The next three sections all refer to the same demo.  You can run the demo yourself at Activity Tabs Demo https://activitytabs.glitch.me and see the source at GoogleChrome/devtools-samples/perf/activitytabs.html .   https://glitch.com/edit/#!/activitytabs?path=index.html -->
+<!-- todo: clean -->
```

Comment:
upstream docs refer to glitch.md for 3 sections.  what's equiv. for this Edge doc?